katello/katello/hostname-change.rb

```diff
@@ -345,12 +346,19 @@ def run
       STDOUT.puts "updating hostname in foreman installer scenarios"
       self.run_cmd("sed -i -e 's/#{@old_hostname}/#{@new_hostname}/g' /etc/foreman-installer/scenarios.d/*.yaml")
 
-      STDOUT.puts "removing last_scenario.yml file"
-      self.run_cmd("rm -rf /etc/foreman-installer/scenarios.d/last_scenario.yaml")
+      if File.exist?(last_scenario_yaml)
+        temp_dir = Dir.mktmpdir
+        STDOUT.puts 'backing up last_scenario.yaml'
+        run_cmd("cp #{last_scenario_yaml} #{temp_dir}")
```

Comment:
Isn't cp + rm equivalent to mv?

Reply:
last_scenario_yaml is a symlink and it seems mv breaks it